src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationCertificationTypeHeader.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <h1 v-if="!isRenewal">{{ `Application for ${certificationType} certification` }}</h1>
-    <h1 v-else>{{ `Application to renew ${certificationType} certification` }}</h1>
+    <h1 v-else>{{ `${certificationType} Renewal` }}</h1>
```

Comment:
Hey see requirement on https://eccbc.atlassian.net/browse/ECER-2358. I _think_ we use this component in application summary and in the application wizard. So this change would break the requirement in ticket ECER-2358. Do you have a requirement in your ticket that is conflicting? If so we should discuss keeping it consistent with design...

Reply:
Ahh really good catch, lets discuss with Zohreh at the standup. We should keep it consistent.